src/test/kotlin/com/michaelsgroi/baseballreference/BrWarDailyBatLinesTest.kt

```diff
@@ -4,19 +4,19 @@ import kotlin.test.Test
 
 class BrWarDailyBatLinesTest {
 
-    private val testee: BrWarDailyBatLines = BrWarDailyBatLines()
+    private val testee = BrWarDailyLines(BrWarDaily.warDailyBatFile, BrWarDaily.SeasonType.BATTING)
 
     @Test
     fun testTopCareerWars() {
-        val playersByWarDescending = testee.getBatterCareers().sortedByDescending { it.battingWar() }
+        val playersByWarDescending = testee.getCareers().sortedByDescending { it.battingWar() }
         val topPlayersByWar =
-            playersByWarDescending.take(5).associate { it.battingWar() to it.playerName }.toSortedMap(reverseOrder())
+            playersByWarDescending.take(5).associate { it.playerName to it.battingWar() }
```

Comment:
sorting on line 11 is key; sorting on line 13 doesn't do much, since this is compared with a(n unsorted) map

Reply:
key and value swapped since player names should never overlap (or is a potential disambiguator needed?), but WARs could—and if two key-value pairs with the same key are added to a map, only the last one is actually kept...